jekyll/_cci2/github-apps-integration.adoc

```diff
@@ -227,12 +219,12 @@ Deploy keys and user keys are not used by GitHub App integrations. All keys are
 [#test-insights]
 === Test Insights
 
-xref:insights-tests#[Test Insights] is currently not supported.
+xref:insights-tests#[Test Insights] is currently not supported for GitHub App integrations.
```

Comment:
Does support for test insights depend on the type of org or the type of pipeline? I'm not sure, but the current phrasing is ambiguous.

Reply:
Ah yeah, org I think, so Test insights is only supported for OAuth orgs. I'll change this. It's tricky because this is the page for people using the GH App integration so that's ambiguous in itself now we have hybrid